applications/tasks/src/modules/kt_rates.erl

```diff
@@ -296,7 +296,7 @@ generate_row([?VARS], Props) ->
     Update = [{<<"pvt_type">>, <<"rate">>}
              ,{<<"rate_name">>, maybe_generate_name(Name, Prefix, ISO, Direction)}
              ,{<<"weight">>, maybe_generate_weight(Weight, Prefix, Cost, IntCost)}
-             ,{<<"routes">>, [<<"^\\+?", Prefix/binary, ".+$">>]}
+             ,{<<"routes">>, [<<"^\\+?", Prefix/binary, ".+\$">>]}
```

Comment:
I didn't want to go this route because it is confusing and I, frankly, think your formatter is broken.

I use the exact same emacs function to format Erlang buffers automatically when saving them:

```emacs-lisp
(add-hook 'erlang-mode-hook                             
          (lambda ()                     
            (add-hook 'before-save-hook 'erlang-indent-current-buffer nil 'make-it-local)))
```
AFAICT, your [fmt.el](https://github.com/fenollp/erlang-formatter/blob/master/fmt.el#L14) does the same thing. I diffed my erlang.el against your repo's version and they are identical. Something else is causing this; maybe emacs 24 (I'm using 25), maybe you're not bringing along enough elisp to do this? Perhaps edts is helping fix this issue in my installation and raw erlang.el has the issue?

Not sure, but I think this commit should be backed out and the formatter fixed.